src/shared/services/configuration/endpoints.js

```diff
@@ -0,0 +1,10 @@
+import isNil from 'lodash.isnil';
```

Comment:
Do we really need an external package to check if something is `null` or `undefined`?